src/arrayMethodSort.js

```diff
@@ -4,8 +4,53 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareAsString(a, b) {
+    if (a == null || b == null) {
+      return -1;
+    }
+
+    let x = a;
+    let y = b;
+
+    x = String(x);
+    y = String(y);
+
+    if (x > y) {
+      return 1;
+    }
+
+    if (x < y) {
+      return -1;
+    }
+
+    return 0;
+  }
+
+  [].__proto__.sort2 = function (compareFunction = compareAsString) {
+    if (typeof compareFunction === 'function') {
+      let count = 0;
```

Comment:
you dont need it here, just declare your variable where you setting value (34 line)